src/components/step1/WalletProviders.tsx

```diff
@@ -1,24 +1,53 @@
 // eslint-disable-next-line
 import React from 'react'
+import styled from 'styled-components'
 import { Provider } from './Provider'
-import { IProviderUserOptions, ThemeColors } from 'web3modal'
+import { IProviderUserOptions } from 'web3modal'
+import { Header2, Paragraph } from '../shared/Typography'
+import { PROVIDERS_WRAPPER_CLASSNAME, ANCHOR_CLASSNAME, PROVIDERS_FOOTER_TEXT_CLASSNAME } from '../../constants/cssSelectors'
 
 interface IWalletProvidersProps {
-  themeColors: ThemeColors
   userOptions: IProviderUserOptions[]
 }
 
-export const WalletProviders = ({ themeColors, userOptions }: IWalletProvidersProps) => <>
-  {userOptions.map(provider =>
-    provider ? (
-      <Provider
-        key={provider.name}
-        name={provider.name}
-        logo={provider.logo}
-        description={provider.description}
-        themeColors={themeColors}
-        onClick={provider.onClick}
-      />
-    ) : null
-  )}
+const ProvidersWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  cursor: pointer;
+  padding: 8px;
+`
+
+const NoWalletAnchor = styled.a`
+  color: #008FF7;
+  text-decoration: none;
+  :hover {
+    color: #4386c6;
+  }
+`
+
+export const WalletProviders = ({ userOptions }: IWalletProvidersProps) => <>
```

Comment:
We can pass down just the providers here, no?

Reply:
`userOptions` are the providers, this is how it was originally named. I'll update the variable name to make it more clear.